### ISSUE TITLE
Unable to download video higher than 24fps.

### ISSUE THREAD

**Issue by Zenodeon** [StefanLobbenmeier/youtube-dl-gui#23]
While downloading the video, I can't see any options that have higher than 24fps even when the video is 60fps.

**Comment by StefanLobbenmeier** [comment]
Yeah that is really annoying, the format selector is a bit broken: #11

**Comment by StefanLobbenmeier** [comment]
Can you link a video where 60 fps is possible? Then I will check the output of yt-dlp
